docs/source/notes/FAQ.md

```diff
@@ -0,0 +1,41 @@
+# Frequently Asked Questions
+
+We list some common problems encountered by users and their corresponding solutions here. Feel free to enrich the lis if you find any frequent issues and have ways to help others to solve them.
+
+## Training
+
+- "Loss goes NaN or very large"
+  1. Check if the dataset annotations are valid. Mask must be `{0, 1}` where `1` for tokens that are **not masked** and `0` for tokens that are **masked**.
+  2. Check `initializer_range` in config file. It can be safely set to `0.02` nn most cases. If the model is very large, decreasing it is a good choice. For example, `initializer_range` can be set to `0.006` when training 175 billion parameter configuration GPT-3 model.
```

Comment:
```suggestion
  2. Check `initializer_range` in config file. It can be safely set to `0.02` in most cases. If the model size is very large, decreasing `initializer_range` is a good choice. For example, `initializer_range` can be set to `0.006` when training 175 billion parameter configuration GPT-3 model.
```
感觉这样合适一些~